src/Categories/Category/Construction/Monoids.agda

```diff
@@ -0,0 +1,74 @@
+{-# OPTIONS --without-K --safe #-}
+open import Categories.Category.Core
+open import Categories.Category.Monoidal.Core
+
+module Categories.Category.Construction.Monoids {o ℓ e} {𝒞 : Category o ℓ e} (C : Monoidal 𝒞) where
+
+open import Level
+
+open import Categories.Functor using (Functor)
+open import Categories.Object.Monoid C
+
+open Category 𝒞
+open Monoidal C
+open HomReasoning
+open Monoid using (η; μ)
+open Monoid⇒
+
+Monoids : Category (o ⊔ ℓ ⊔ e) (ℓ ⊔ e) e
+Monoids = record
+  { Obj = Monoid
+  ; _⇒_ = Monoid⇒
+  ; _≈_ = λ f g → arr f ≈ arr g
+  ; id = λ {A} → record
+    { arr = id
+    ; preserves-μ = begin
+      id ∘ μ A
```

Comment:
We prefer to use side-by-side alignment whenever possible.

Reply:
Have updated it to use this style.